app/src/main/java/com/example/geoquiz/MainActivity.kt

```diff
@@ -54,14 +47,16 @@ class MainActivity : AppCompatActivity() {
         }
 
         cheatButton.setOnClickListener { view ->
-            val answerIsTrue = quizViewModel.currentQuestionAnswer
-            val intent = CheatActivity.newIntent(this@MainActivity, answerIsTrue)
+            val answerQuestion = quizViewModel.currentQuestionAnswer
+            val intent = CheatActivity.newIntent(this@MainActivity, answerQuestion)
             if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
When things become crowded inside lambda brackets, that's a sign that code can be moved into a function. In this case I would add the if statement inside a `openCheatActivity(intent: Intent)` function